packages/grid/x-data-grid/src/locales/urPK.ts

```diff
@@ -0,0 +1,152 @@
+import { urPK as urPKCore } from '@mui/material/locale';
+import { GridLocaleText } from '../models/api/gridLocaleTextApi';
+import { getGridLocalization, Localization } from '../utils/getGridLocalization';
+
+const urPKGrid: Partial<GridLocaleText> = {
```

Comment:
Should probably work fine as it seems to have all the translations provided. 🤔 
```suggestion
const urPKGrid: GridLocaleText = {
```

Reply:
We can't because of this [key](https://github.com/mui/mui-x/blob/8225f53c5a02424b7f683a45c4ca9516bfa1a40b/packages/grid/x-data-grid/src/models/api/gridLocaleTextApi.ts#L150-L153) added to GridLocaleText, this is to allow passing custom locales (including MuiTablePagination) from outside using `localeText` prop as explained [here](https://mui.com/x/react-data-grid/localization/#translation-keys).

Reply:
Ah, didn't look into it, fair point. 👌